gitindex/index.go

```diff
@@ -554,20 +554,33 @@ func indexGitRepo(opts Options, config gitIndexConfig) error {
 			if err != nil {
 				return err
 			}
+
+			var pathRank []float64
+			if rank, ok := ranks.Paths[keyFullPath]; ok {
+				pathRank = []float64{rank}
+			}
+			fmt.Printf("path: %s, ranks length: %d", keyFullPath, len(pathRank))
```

Comment:
looks like a sneaky debug statement got through

Reply:
Oops, thanks for catching this!